Cargo.toml

```
@@ -156,3 +156,6 @@ zip32 = "0.1.1"
 lto = true
 panic = 'abort'
 codegen-units = 1
+
+[lints.rust]
```

`[lints]` sections are not valid in workspace manifests. This should instead be:
```suggestion
[workspace.lints.rust]
```